controllers/feeds/getFeedsV2.js

```diff
@@ -249,6 +256,13 @@ module.exports = async (req, res) => {
         });
       }
     }
+    // get karma score for each post actor
+    const karmaScores = await UsersFunction.getUsersKarmaScore(User, postActors);
+    for (let i = 0; i < data.length; i++) {
+      const user = karmaScores.find((user) => user.user_id === data[i].actor.id);
+      data[i].karma_score = user?.karma_score || 0;
+      data[i] = await modifyAnonimityPost(data[i], isBlurredPost);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for getting karma scores for each post actor and assigning them to the corresponding posts seems fine. However, ensure that the function `UsersFunction.getUsersKarmaScore` handles any potential errors and does not cause the entire loop to fail.


<!-- This is an auto-generated comment by OSS CodeRabbit -->